CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.0.3 / 2021-08-17
+==================
+- Fixed `EventEmitter` memory leak (see issue [#81](https://github.com/apify/proxy-chain/issues/81))
+- Added automated tests for Node 16
+- Updated dev dependencies
```

Comment:
🤔 Did you forget to push the commit?

Reply:
It was dependabot PRs merged directly to master before

Reply:
changelog is for end users, and they dont care about dev deps in general

(but let's keep it here, it won't hurt anyone)